cmd/batch.go

```diff
@@ -356,7 +411,7 @@ func (b *standardBatchPipeline) WriteFile(
 		return nil
 	}
 
-	return b.WriteTemplate(writer, string(fileBytes), true, templateData)
+	return b.WriteTemplate(writer, string(fileBytes), pregapLines+1, postgapLines, templateData)
```

Comment:
Lets not add one here, lets add it to the write file calls directly, that way its a little more explicit on where the plus ones come from and gives us more control if we need to later on.